### ISSUE TITLE
Install broken, source missing

### ISSUE THREAD

**Issue by authago** [YunoHost-Apps/wekan_ynh#13]
It seems the Wekan install in Yunohost is broken. From the logs I understand that the source can't be downloaded from 'https://releases.wekan.team' any more. It looks like it was pushed out of the page since the app moved on to newer versions.

```
2018-04-18 20:57:33,196 INFO     yunohost.hook <lambda> - [568.88] + src_format=tar.gz
2018-04-18 20:57:33,196 INFO     yunohost.hook <lambda> - [568.88] + '[' wekan-0.77.tar.gz = '' ']'
2018-04-18 20:57:33,196 INFO     yunohost.hook <lambda> - [568.88] + local local_src=/opt/yunohost-apps-src/wekan/wekan-0.77.tar.gz
2018-04-18 20:57:33,196 INFO     yunohost.hook <lambda> - [568.88] + test -e /opt/yunohost-apps-src/wekan/wekan-0.77.tar.gz
2018-04-18 20:57:33,196 INFO     yunohost.hook <lambda> - [568.88] + wget -nv -O wekan-0.77.tar.gz https://releases.wekan.team/wekan-0.77.tar.gz
2018-04-18 20:57:33,399 WARNING  yunohost.hook <lambda> - [568.88] https://releases.wekan.team/wekan-0.77.tar.gz:
2018-04-18 20:57:33,399 WARNING  yunohost.hook <lambda> - [568.88] 2018-04-18 20:57:33 Fout 404: Not Found.
2018-04-18 20:57:33,400 INFO     yunohost.hook <lambda> - [568.88] + ynh_exit_properly
2018-04-18 20:57:33,401 INFO     yunohost.hook <lambda> - [568.88] + local exit_code=8
```

I tried placing a download from Github local as /opt/yunohost-apps-src/wekan/wekan-0.77.tar.gz but then the checksum fails, so I must have the wrong file:

```
2018-04-18 21:06:36,457 INFO     yunohost.hook <lambda> - [568.102] + local local_src=/opt/yunohost-apps-src/wekan/wekan-0.77.tar.gz
2018-04-18 21:06:36,457 INFO     yunohost.hook <lambda> - [568.102] + test -e /opt/yunohost-apps-src/wekan/wekan-0.77.tar.gz
2018-04-18 21:06:36,457 INFO     yunohost.hook <lambda> - [568.102] + cp /opt/yunohost-apps-src/wekan/wekan-0.77.tar.gz wekan-0.77.tar.gz
2018-04-18 21:06:36,457 INFO     yunohost.hook <lambda> - [568.102] + echo '4d911547504fd6388779faf9f79cc4e527c2f2b4f40b8b7b82ad05783e2cd0a3 wekan-0.77.tar.gz'
2018-04-18 21:06:36,457 INFO     yunohost.hook <lambda> - [568.102] + sha256sum -c --status
2018-04-18 21:06:36,458 INFO     yunohost.hook <lambda> - [568.102] + ynh_die 'Corrupt source'
2018-04-18 21:06:36,458 WARNING  yunohost.hook <lambda> - [568.102] Corrupt source
2018-04-18 21:06:36,458 INFO     yunohost.hook <lambda> - [568.102] + echo 'Corrupt source'
2018-04-18 21:06:36,458 INFO     yunohost.hook <lambda> - [568.102] + exit 1
2018-04-18 21:06:36,458 WARNING  yunohost.hook <lambda> - [568.102] !!
2018-04-18 21:06:36,458 INFO     yunohost.hook <lambda> - [568.102] + ynh_exit_properly
2018-04-18 21:06:36,458 WARNING  yunohost.hook <lambda> - [568.102]   wekan's script has encountered an error. Its execution was cancelled.
```

**Comment by alexAubin** [comment]
Ah, thanks for the feedback. That's a bit annoying that those archive suddenly disappear :/ 

Yesterday, anmol opened a pull request to update the source file, but it didnt get tested so far.

If you want, you can try it. It should be as easy as running : 

```
yunohost app install https://github.com/anmol26s/wekan_ynh
```

**Comment by 4abhinavjain** [comment]
Yea I noticed the same problem. Was able to install it through anmol's link.

**Comment by alexAubin** [comment]
Should be fixed now that #12 is merged :wink:

**Comment by anmol26s** [comment]
This issue has started again. Isn't it better to use github release https://github.com/wekan/wekan/releases ?
